README.md

```diff
@@ -1,27 +1,21 @@
+[![Build Status](https://travis-ci.com/HumanCompatibleAI/airl.svg?branch=master)](https://travis-ci.com/HumanCompatibleAI/airl)
+
 # Adversarial Inverse Reinforcment Learning
 
 This repository reimplements AIRL with modern dependencies.
 Paper: https://arxiv.org/abs/1710.11248
 Original Implementation: https://github.com/justinjfu/inverse_rl
 
 To install:
-
 ```
-conda create -n yairl python=3.6.8  # Tensorflow doesn't work with python 3.7 at the time of writing.
+sudo apt install libopenmpi-dev
```

Comment:
Consider making a setup.sh that does all of this.

Reply:
I feel this is sufficiently platform dependent (e.g. people might need to use `yum` rather than `apt`, `virtualenv` rather than `conda`) I'd rather not make it a script. However, we should definitely consider making a script if it makes our life easier e.g. for testing or deployment (although I'd probably favor Docker at that point).